puck.config.tsx

```diff
@@ -30,6 +33,7 @@ export const config: Config<Props> = {
         </div>
       ),
     },
```

Comment:
Since you created the component can you please also remove the HeadingBlock component here. It's no longer needed.